routing/routing_suite_test.go

```diff
@@ -90,49 +104,37 @@ func GetAppGuid(appName string) string {
 	return strings.TrimSpace(string(appGuid))
 }
 
-func MapRouteToApp(domain, path, app string) {
-	spaceGuid, domainGuid := GetSpaceAndDomainGuids(app)
+func MapRouteToApp(app, domain, host, path string) {
+	Expect(cf.Cf("map-route", app, domain, "--hostname", host, "--path", path).Wait(DEFAULT_TIMEOUT)).To(Exit(0))
+}
 
-	routeGuid := CreateRoute(domain, path, spaceGuid, domainGuid)
-	appGuid := GetAppGuid(app)
+func DeleteRoute(hostname, contextPath, domain string) {
+	Expect(cf.Cf("delete-route", domain,
+		"--hostname", hostname,
+		"--path", contextPath,
+		"-f",
+	).Wait(DEFAULT_TIMEOUT)).To(Exit(0))
+}
 
-	Expect(cf.Cf("curl", "/v2/apps/"+appGuid+"/routes/"+routeGuid, "-X", "PUT").Wait(CF_PUSH_TIMEOUT)).To(Exit(0))
+func CreateRoute(hostname, contextPath, space, domain string) {
+	Expect(cf.Cf("create-route", space, domain,
+		"--hostname", hostname,
+		"--path", contextPath,
+	).Wait(DEFAULT_TIMEOUT)).To(Exit(0))
 }
 
-func CreateRoute(hostName, contextPath, spaceGuid, domainGuid string) string {
-	jsonBody := "{\"host\":\"" + hostName + "\", \"path\":\"" + contextPath + "\", \"domain_guid\":\"" + domainGuid + "\",\"space_guid\":\"" + spaceGuid + "\"}"
-	session := cf.Cf("curl", "/v2/routes", "-X", "POST", "-d", jsonBody).Wait(CF_PUSH_TIMEOUT)
-	routePostResponseBody := session.Out.Contents()
+func GetRouteGuid(hostname, path string) string {
```

Comment:
Does this method need to be exported? I did a quick search and it doesn't look like it's used outside the package:

```
± |master ↑2 ✓| → ag GetRoute
routing/route_services_test.go
159:	routeGuid := GetRouteGuid(hostname, "")
173:	routeGuid := GetRouteGuid(hostname, "")
208:	routeGuid := GetRouteGuid(hostname, "")

routing/routing_suite_test.go
126:func GetRouteGuid(hostname, path string) string {
```